packages/scandipwa/src/component/Header/Header.style.scss

```diff
@@ -383,6 +383,20 @@
 
     &_isCheckout {
         .Header {
+            &-Nav {
+                display: flex;
+            }
+
+            &-IconsWrapper {
+                display: flex;
+                width: 89.4%;
```

Comment:
Very strange width value

Reply:
When I measured the icon position with page ruler that was the exact value to align the position. 90% was a bit right for the account icon, so that's why I left the value like that.